python/graphstorm/trainer/np_trainer.py

```diff
@@ -167,9 +168,10 @@ def fit(self, train_loader, num_epochs,
                 if self.evaluator is not None and \
                     self.evaluator.do_eval(total_steps, epoch_end=False) and \
                     val_loader is not None:
+
                     val_score = self.eval(model.module if is_distributed() else model,
```

Comment:
Can we modify the eval() function instead of the fit() function?
And we should add an warning here that we force the return_proba = True.